template.yaml

```diff
@@ -787,6 +836,30 @@ Resources:
         - Key: Name
           Value: !Sub 'SNSTopicCacheAPI-${DeployStage}'
 
+  SNSStudyPayload:
+    Type: AWS::SNS::Topic
+    Properties:
+      ContentBasedDeduplication: True
+      FifoTopic: True
```

Comment:
This addition does two things:
- insures that a message won't be delivered twice
- insures delivery order to all consumers